addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -22,6 +22,13 @@
 				<label>Address</label>
 				<description>The address of the AVR to control.</description>
 			</parameter>
+			<parameter name="PORT" type="integer" required="false">
+				<context>network-address</context>
```

Comment:
This one still has to be removed

Reply:
@martinvw Sorry I missed this one. What exactly has to be done here? Should the line `<context>network-address</context>` be simply removed?

Reply:
Yes the comment got outdated because of the reformatting. But this content is meant for host names not for ports

Reply:
@martinvw this is now fixed. Feel free to review the amended commit.